content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -38,11 +39,12 @@ If a {{< glossary_tooltip term_id="node" >}} dies, the Pods scheduled to that no
 are [scheduled for deletion](#pod-garbage-collection) after a timeout period.
 
 Pods do not, by themselves, self-heal. If a Pod is scheduled to a
-{{< glossary_tooltip text="node" term_id="node" >}} that then fails, the Pod is deleted; likewise, a Pod won't
-survive an eviction due to a lack of resources or Node maintenance. Kubernetes uses a
-higher-level abstraction, called a
-{{< glossary_tooltip term_id="controller" text="controller" >}}, that handles the work of
-managing the relatively disposable Pod instances.
+{{< glossary_tooltip text="node" term_id="node" >}} that then fails, the Pod is
+deleted; likewise, a Pod won't survive an eviction due to a lack of resources or
+Node maintenance. What Pods can do is self-heal on a local node. In order to have
+self-healing across the cluster, Kubernetes uses a higher-level abstraction, called
```

Comment:
To me, self-healing explicitly says that the Pod controls its own health. I don't think that's accurate though, it's a controller than manages the Pod lifecycle and not the Pod itself that manages it. 

Maybe ask someone from @kubernetes/sig-node-pr-reviews or a more experienced docs reviewer for confirmation, but I feel like this muddles the clarity a teeny bit

Reply:
yeah this muddies things a little for me too.